src/libcore/str/pattern.rs

```diff
@@ -1178,8 +1178,8 @@ impl TwoWaySearcher {
 trait TwoWayStrategy {
     type Output;
     fn use_early_reject() -> bool;
-    fn rejecting(usize, usize) -> Self::Output;
-    fn matching(usize, usize) -> Self::Output;
+    fn rejecting(a: usize, b: usize) -> Self::Output;
```

Comment:
I tried not to be creative and used names already used by the implementations of the trait. I didn't give too much thought to this particular example as it is actually a private trait and does not appear in the docs like the others.